### PR TITLE
Get obal to work with pulp's nightly process

### PR DESCRIPTION
**Pull request by pcreech**
Get obal to work with pulp's nightly process

### CODE REVIEW

File: obal/data/roles/setup_sources/tasks/git.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+- name: 'Set source checkout location'
+  set_fact:
+    setup_sources_git_checkout_dir: "/tmp/ansible_git_{{ inventory_hostname }}"
+
+- name: 'Ensure git location'
+  file:
```
Comment (ehelms) [inline obal/data/roles/setup_sources/tasks/git.yml]: Does the git module not create the directory by default?

File: obal/data/roles/setup_sources/tasks/main.yml [hunk 2]
```diff
@@ -5,28 +5,14 @@
...
     - always
 
-- name: get spec file
-  find:
-    pattern: "*.spec"
-    path: "{{ package_dir }}"
-  register: spec_file
+- name: 'Setup sources from specfile'
+  include_tasks: specfile.yml
+  when: git is not defined
```
Comment (ehelms) [inline obal/data/roles/setup_sources/tasks/main.yml]: Should this be `setup_sources_git` ?

File: obal/data/roles/setup_sources/tasks/git.yml (hunk 1 above)
Comment (pcreech) [reply obal/data/roles/setup_sources/tasks/git.yml]: yup, this was a relic, removed

File: obal/data/roles/setup_sources/tasks/main.yml (hunk 2 above)
Comment (pcreech) [reply obal/data/roles/setup_sources/tasks/main.yml]: Yup, relic from testing with pulp's current nightly code.  fixed

**Pull request merged**